special-interest-groups/sig-k8s/member-list.md

```diff
@@ -0,0 +1,18 @@
+# SIG K8S Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2020-02-21
+
+## Tech Leads
```

Comment:
there are too many leads, could you limit the number to no more than 3?

Reply:
updated